.github/workflows/syncRootfs.ps1

```diff
@@ -0,0 +1,85 @@
+<#
+.SYNOPSIS
+    Sync your WSL tarball to cloud-images.ubuntu.com/wsl
+.DESCRIPTION
+    Compare checksums with the remote tarball and download it if the checksums don't match.
+.PARAMETER Path
+    The path to your local copy of the rootfs tarball.
+.PARAMETER CodeName
+    The codename of the release you want to sync it to (e.g: jammy, kinetic, lunar).
+.EXAMPLE
+    PS> .\syncRootfs.ps1 -Path $HOME\Downloads\jammy.tar.gz -CodeName jammy
+
+    This downloads the latest Ubuntu 22.04 LTS image into your downloads folder. If you run it
+    again, the rootfs will not be downloaded unless the remote is updated or your local file is
+    damaged.
+.NOTES
+    Author: Edu Gómez Escandell
+#>
+
+# Download latest LTS rootfs
+param(
+    [Parameter(Mandatory, HelpMessage="The path wou want your rootfs in.")][string]$Path,
+    [Parameter(Mandatory, HelpMessage="The codename of the release of your rootfs.")][string]$CodeName
+)
+
+$rootfsDir = $(Split-Path -Path "${Path}")
+$tarball = $(Split-Path -Path "${Path}" -Leaf)
+$tmpDir = "C:\Temp\syncRootfs-${CodeName}"
+
+# This function validates a rootfs against its checksum
+function Test-Rootfs {
+    param ([Parameter(Mandatory)][string]$Path)
+
+    $sha256file = "${tmpDir}\${CodeName}.SHA256SUMS.txt"
+
+    if ($(Test-Path -Path "${Path}")) {
```

Comment:
(you can use an early return to avoid this identation, but this is not Go, so as you wish :))

Reply:
Godd catch :P. Fixed.